content/projects/jenkins-operator/index.adoc

```diff
@@ -0,0 +1,42 @@
+---
+layout: project
+title: "Jenkins Operator"
+section: projects
+tags:
+- kubernetes
+- infrastructure
+links:
+  github: 'jenkinsci/kubernetes-operator'
+  slack: 'https://virtuslab-oss.slack.com/'
+---
+
+The Jenkins Operator is a link:https://kubernetes.io/docs/concepts/extend-kubernetes/operator/[*Kubernetes Native Operator*] which manages operations for Jenkins on Kubernetes. It has been built with Immutability and declarative Configuration as Code in mind.
+
+== Project Vision
+
+With Jenkins Operator project we want to enable community to run Jenkins in cloud-native environments like Kubernetes. Also, support most of the public cloud providers (AWS, Azure, GCP) in terms of additional capabilities like backups, observability and cloud security.
+
+With declarative configuration and full lifecycle management based on Operator Framework this can become the de facto standard for running Jenkins on top of Kubernetes.
```

Comment:
```suggestion
With declarative configuration and full lifecycle management based on the link:https://operatorframework.io/[Operator Framework] this can become the de facto standard for running Jenkins on top of Kubernetes.
```